pkg/vmimage/data/script.sh

```diff
@@ -36,7 +36,7 @@ cat >client-key.pem <<'EOF'
 {{ .Builder.ClientKey | PrivateKeyAsBytes | String }}
 EOF
 
-go get github.com/jim-minter/tlsproxy
+go get github.com/openshift/openshift-azure/cmd/
 go/bin/tlsproxy -insecure -key client-key.pem -cert client-cert.pem https://cdn.redhat.com/ &
```

Comment:
This needs to change to smth like `github.com/openshift/openshift-azure/cmd/azure tlsproxy` (need to doublecheck syntax)